Index: src/Main.java
===================================================================
diff --git a/src/Main.java b/src/Main.java
deleted file mode 100644
--- a/src/Main.java	(revision e8f2ff2d53baa48d17ed4a06b4f4c83a1c20b258)
+++ /dev/null	(revision e8f2ff2d53baa48d17ed4a06b4f4c83a1c20b258)
@@ -1,5 +0,0 @@
-public class Main {
-    public static void main(String[] args) {
-        System.out.println("Hello, Visual Studio Code!");
-    }
-}
Index: README.md
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/README.md b/README.md
deleted file mode 100644
--- a/README.md	(revision e8f2ff2d53baa48d17ed4a06b4f4c83a1c20b258)
+++ /dev/null	(revision e8f2ff2d53baa48d17ed4a06b4f4c83a1c20b258)
@@ -1,1 +0,0 @@
-# Byte-Cast-Productivity-App
\ No newline at end of file
